Makefile

```diff
@@ -1,28 +1,29 @@
-.PHONY: build compile run
+.PHONY: dependencies
+COMPONENTNAME = mozart-fetcher
+REGION = eu-west-1
+BUILDPATH = /root/rpmbuild
 
-build:
-	docker build -t mozart_fetcher .
+none:
+	@ echo Please specifiy a target
+
+dependencies:
+	mix deps.get
 
-compile:
-	docker run \
-		--rm=true \
-		-v $(CURDIR):/opt/app \
-		-w /opt/app \
-		-e "DEV_CERT_PEM=$(DEV_CERT_PEM)" \
-		-e "APP_NAME=mozart_fetcher" \
-		-e "APP_VERSION=0.1.0" \
-		-e "RELEASE_DIR=_build/prod/rel/mozart_fetcher/releases/0.1.0/" \
-		-p 8080:8080 \
-		mozart_fetcher sh -c 'rm -rf _build tars && mkdir -p tars && mix release && cp _build/prod/rel/mozart_fetcher/releases/0.1.0/mozart_fetcher.tar.gz /opt/app/tars/'
+test:
+	MIX_ENV=test mix test
+
+build:
+	$(eval COSMOS_VERSION:=$(shell cosmos-release generate-version ${COMPONENTNAME}-${REGION}))
+	mix distillery.release
+	mkdir -p ${BUILDPATH}/SOURCES
+	cp _build/prod/rel/mozart_fetcher/releases/*/mozart_fetcher.tar.gz ${BUILDPATH}/SOURCES/
+	tar -zcf ${BUILDPATH}/SOURCES/bake-scripts.tar.gz bake-scripts/
+	cp mozart-fetcher.spec ${BUILDPATH}/SOURCES/
+	cp SOURCES/* ${BUILDPATH}/SOURCES/
+	rpmbuild --define "_topdir ${BUILDPATH}" --define "version ${COSMOS_VERSION}" --define '%dist .bbc.el8' -ba mozart-fetcher.spec
 
-run:
-	docker run \
-		--rm=true \
-		-v $(CURDIR):/opt/app \
-		-v $(DEV_CERT_PEM):$(DEV_CERT_PEM) \
-		-w /opt/app \
-		-e "DEV_CERT_PEM=$(DEV_CERT_PEM)" \
-		-e "APP_NAME=mozart_fetcher" \
-		-e "APP_VERSION=0.1.0" \
-		-p 8080:8080 \
-		mozart_fetcher sh -c 'mix run --no-halt'
+release:
+	echo "Releasing 'RPMS/**/*.rpm' to ${COMPONENTNAME}-${REGION}"
+	cosmos-release service "${COMPONENTNAME}-${REGION}" --release-version=v ${BUILDPATH}/RPMS/x86_64/*.x86_64.rpm
+	cosmos-release service "${COMPONENTNAME}-weather-${REGION}" --release-version=v ${BUILDPATH}/RPMS/x86_64/*.x86_64.rpm
+	cosmos-release service "${COMPONENTNAME}-sport-${REGION}" --release-version=v ${BUILDPATH}/RPMS/x86_64/*.x86_64.rpm
```

Comment:
Only thing I'd maybe suggest is doing something like this:
https://github.com/bbc/mozart-controller/pull/621/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R37-R40
requires setting this variable for the build in Codebuild

Reply:
Done here with loop https://github.com/bbc/mozart-fetcher/pull/74/commits/f14d35fd69b70ff25958458a13223a8cfa37f27c, but I keep the `COMPONENTS` define in the make file instead of ENV in codebuild.

Reply:
I actually could not make it work I get some syntax error or some evaluation error with the name of the component
e.g. 
with this version
```shell
for component in "mozart-fetcher-eu-west-1 mozart-fetcher-weather-eu-west-1 mozart-fetcher-sport-eu-west-1"; do \
done
```
I am getting following error
```shell
ENOENT: no such file or directory, open 'mozart-fetcher-weather-eu-west-1'
--
<span class="ace_error">make: *** [Makefile:28: set_repositories] Error 1</span>
```
I decide to move on and have this version https://github.com/bbc/mozart-fetcher/pull/74/commits/cc26c5f7d105884f49e67c3a3ac62d9c0cb7b7f1. I could not make it work after trying 2 hours.

Reply:
@JoeARO did you manage to have a version that works with the `for X in XS` syntax?

Reply:
https://github.com/bbc/mozart-controller/pull/621/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R33-R40 works

you need \ cause otherwise it thinks each line is a command iirc.

Reply:
Kinda hard to know what the error was without the rest of code too though

Reply:
I did that, but still get error like the `'mozart-fetcher-weather-eu-west-1'` is a file not ilke command params in the case name of the component to witch want to push the repositories.json

Reply:
@JoeARO I add it and to the release here https://github.com/bbc/mozart-fetcher/pull/74/commits/ab70a892a6d4490b37ffedd0a961c5a835396ff2 and remove the COMPONENTS from the build file https://github.com/bbc/mozart-fetcher/pull/74/commits/84cfb5e748fa44631bbbf0038b2d7c0c60388be6